src/util/helpers/helpers.cpp

```diff
@@ -417,18 +417,19 @@ std::string GenerateRandomString(size_t length)
 	return GenerateRandomString(length, kCharacters);
 }
 
-std::string GenerateRandomString(size_t length, std::string_view characters)
+std::string GenerateRandomString(const size_t length, const std::string_view characters)
 {
 	assert(!characters.empty());
-	std::stringstream result;
+	std::string result;
+	result.reserve(length);
```

Comment:
This can be made even more efficient with two simple changes. The first is to use .resize here

Reply:
Done

Reply:
Just curious, how does `.resize()` improve things compared to `.reserve()`?